src/ImageCarouselReact/widget/components/Glyphicon.js

```diff
@@ -16,10 +16,13 @@ define(["require", "exports", "ImageCarouselReact/lib/classnames", "ImageCarouse
             logger.debug(this.loggerNode + " .render");
```

Comment:
please ignore all .js  file changes.
Files derived from .ts(x) files should not be in the repo